src/main/sh/mommy

```diff
@@ -192,7 +192,7 @@ fill_template() {
 MOMMY_OPT_HELP=""
 MOMMY_OPT_VERSION=""
 MOMMY_OPT_TARGET="2"
-MOMMY_OPT_CONFIG_FILE="$HOME/.config/mommy/config.sh"
+MOMMY_OPT_CONFIG_FILE=${MOMMY_OPT_CONFIG_FILE:-"${XDG_CONFIG_HOME:-"$HOME/.config"}/mommy/config.sh"}
```

Comment:
...i just remembered, you have to put quotes around this, the default shell on netbsd requires quotes even when assigning a variable, otherwise word splitting might occur. it's weird, i know, but that's just netbsd~

(at least it's not solaris!)

Reply:
awee! done (: